homeassistant/components/plex/server.py

```diff
@@ -39,12 +42,37 @@
 plexapi.X_PLEX_VERSION = X_PLEX_VERSION
 
 
+def debounce(func):
+    """Decorate function to debounce callbacks from Plex websocket."""
+
+    unsub = None
+
+    async def call_later_listener(self, _):
+        """Handle call_later callback."""
+        nonlocal unsub
+        unsub = None
+        await self.hass.async_add_executor_job(func, self)
+
+    @wraps(func)
+    def wrapper(self):
+        """Schedule async callback."""
+        nonlocal unsub
+        if unsub:
+            _LOGGER.debug("Throttling update of %s", self.friendly_name)
+            unsub()  # pylint: disable=not-callable
```

Comment:
Does this mean that as long as updates come in more frequently than the debounce time, the function will never be called?

Reply:
Yes, but the updates are normally infrequent. This is to handle the times when it bursts.